===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(swReactorPoll_create:object->fds)) at 15 (line 57, column 9) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/swoole-src/src/reactor/ReactorPoll.c
+++ b/table1/benchmarks/swoole-src/src/reactor/ReactorPoll.c
@@ -56,6 +56,7 @@ int swReactorPoll_create(swReactor *reactor, int max_fd_num)
     object->events = sw_calloc(max_fd_num, sizeof(struct pollfd));
     if (object->events == NULL)
     {
+        free(object);
         swWarn("malloc[2] failed");
         return SW_ERR;
     }
